### PR TITLE
[Holy] 10.2.6 Changes

### PR DESCRIPTION
**Pull request by Saeldur**
### Description
Make an attempt at overhauling the Holy Talents to be somewhat more up to date. This still does not bring them up to a feature complete state nor guarantees everything is functional, especially changes that were not made in 10.2.6 due to the amount of time the spec has not been maintained for.

Notably, this is still missing the Dragonflight S3 Set Bonus.

Due to how the Dragonflight S3 Set Bonus and Lightwell work with Renew the current implementation is insufficient to properly capture the HPS values of these, for now I have used some approximations for Lightwell, Salvation and Benediction that should still be more accurate than the previous heuristics used.

Ideally, the Renew implementation present needs to be redone entirely with a custom normalizer to fully assign ownership of the renews including in cases where they are extended rather than refreshed (S3 Set, Lightwell extend without log event).

### 10.2.6 Changes Made
- [ ] Empyreal Blaze
- [x] Pontifex Redesign
- [x] Implement Lightwell
- [x] Resonant Words Changes
- [x] Divine Word Changes
- [x] Healing Chorus Numbers Update
- [x] Prayerful Litany Numbers Update
- [ ] Voice of Harmony Full Rename
- [ ] Voice of Harmony Holy Nova for Chastise
- [x] Add a changelog to WoWA documenting the changes

### Other Changes Made
- Update Salvation to claim renews within 15 seconds of Salvation heal events (e.g. on targets it applied renew to). Fully claim this healing for the full duration over other sources of renew like Benediction as these renews will always be active during this window from Salvation.
- Update Benediction to claim Renews rather than using a heuristic which is drastically thrown off by the unimplemented nature of the Dragonflight S3 Tierset which procs renews. This is now more accurate.

### Testing
- https://www.warcraftlogs.com/reports/rC4LwgAGMjTv13Rk#fight=3&type=summary&source=2 =>M+ (Damage Components of Divine Word)
- https://www.warcraftlogs.com/reports/AhtXFpz2TGWgLM9k#fight=1&type=summary&source=18 => Raid, Divine Word Serenity
- https://www.warcraftlogs.com/reports/GtMrCDk1ZVRwmzWH#fight=6&type=summary&source=18  => Raid, Pontifex
- https://www.warcraftlogs.com/reports/8YdLwPjkMmCGvbqF#fight=48&type=healing&source=15 => Raid, Lightwell, Resonant Words, Pontifex

### CODE REVIEW

**Comment by Saeldur** [comment]
Attempting to merge in changes from upstream made github go full Schizo, forcepush reverted >_>'

**Comment by Saeldur** [comment]
Couple of things:

Labels got messed up by a merge gone wrong (sorry)
Not everything is fully implemented (there's a few damage changes from the recent 10.2.6 changes not done re damage spells / internal naming and its missing the Season 3/4 tierset still)

Module accuracy in general is still likely poor due to how dated it is. Voll helped me hack together the linking for the Renews but this really needs to be redone from the groundup with custom linkage to actually form parent child relationships between actions and their casts ideally even tracking POM bounces if possible but that might prove a challenge.

The linkage on the renews for the Tierset (if implemented) and from lightwell will be odd due to their pure extend behaviour which does not trigger any events hence the bodge job. This is going to attribute it slightly incorrectly in terms of the fact that it will assign the renews for the period after the heal hit rather than the period in which was associated with that extension.

It's probably still best if this module triggers a warning on attempting to use it indicating the feature incomplete and potentially inaccurate state but I wanted to update a bunch of it to help out the Holy Priests. Not sure how to make it still trigger a warning other than just bumping the version number down though.

File: src/analysis/retail/priest/holy/modules/talents/BottomRow/Pontifex.tsx [hunk 1]
```diff
@@ -7,43 +7,51 @@ import Events, { HealEvent } from 'parser/core/Events';
...
+  SPELLS.FLASH_HEAL,
+  SPELLS.GREATER_HEAL,
+  TALENTS.PRAYER_OF_HEALING_TALENT,
+  TALENTS.CIRCLE_OF_HEALING_TALENT,
+];
 
 /**
  * Critical heals from Flash Heal and Heal increase your healing done by
  * your next Holy Word spell by 20%, stacking up to 2 times.
  */
```
Comment (emallson) [inline src/analysis/retail/priest/holy/modules/talents/BottomRow/Pontifex.tsx]: is this comment outdated?

**Review by emallson — APPROVED** [review]
minor nit about outdated pontifex comment. the rest looks fine.

(insert oprah meme "you get a renew! and you get a renew! and you get a renew!")

File: src/analysis/retail/priest/holy/modules/talents/BottomRow/Pontifex.tsx (hunk 1 above)
Comment (Saeldur) [reply src/analysis/retail/priest/holy/modules/talents/BottomRow/Pontifex.tsx]: Yep it was the old talent, fixed that.

**Pull request merged**